1-callbacks/solution.js

```diff
@@ -32,16 +32,57 @@ node solution.js name1 name2 name3
 ** give a look to node.js util.promisify, avoid to alter the validate-user.file **
 */
 
+const validator = require('./validate-user');
+
 function solution() {
     // YOUR SOLUTION GOES HERE
 
     // you get your 5 names here
-
+    const vars = process.argv.slice(2).length ? [...process.argv.slice(2)] : ['John', 'Diana', 'Jose', 'Stacy', 'Mary'];
+    let splitNames = [];
+    if (vars[0]?.includes(',')) {
+      splitNames = vars[0].split(',');
+    }
+    const names = splitNames.length ? [...splitNames] : [...vars];
+
+    const failures = [];
+    const success = [];
+
+    let counter = 0;
+
+    const callback = (error, data) => {
+      if(error?.message){
+        failures.push(error.message);
+
+      } else if (data){
+        success.push(data);
+      }
+
+      counter++;
+      // log the final result
+      if(counter === names.length){
```

Comment:
I would just log the array. It doesn't seem very valuable to add all this logic to get the output exactly like the example